lib/response.js

```diff
@@ -822,6 +823,14 @@ res.get = function(field){
  */
 
 res.clearCookie = function clearCookie(name, options) {
+  if (options && (options.maxAge || options.expires)) {
```

Comment:
I dont think we need to do this check twice right? This is totally a nitpick and not required before landing, but it feels weird to me to use one block checking for either/or, then to also do a more specific check for each inside it.

Reply:
Good point, there's no reason to check twice.

This was overengineered.